src/person/views.py

```diff
@@ -0,0 +1,79 @@
+import json
+
+from app import settings as api_settings
+from app.utils import success_response
+from django.contrib.auth.models import User
+from rest_framework import generics
+from rest_framework import status
+
+from .controllers.authenticate_controller import AuthenticateController
+from .controllers.delete_controller import DeletePersonController
+from .controllers.developer_controller import DeveloperController
+from .controllers.update_controller import UpdatePersonController
+from .serializer import AuthenticateSerializer
+from .serializer import UserSerializer
+
+
+class AuthenticateView(generics.GenericAPIView):
+    serializer_class = AuthenticateSerializer
+    permission_classes = api_settings.CONSUMER_PERMISSIONS
+
+    def post(self, request):
+        """Authenticate the current user."""
+        try:
+            data = json.loads(request.body)
+        except json.JSONDecodeError:
+            data = request.data
+        return AuthenticateController(request, data, self.serializer_class).process()
+
+
+class MeView(generics.GenericAPIView):
+    serializer_class = UserSerializer
+    permission_classes = api_settings.CONSUMER_PERMISSIONS
+
+    def get(self, request):
+        """Get current authenticated user."""
+        return success_response(
+            self.serializer_class(request.user).data, status.HTTP_200_OK
+        )
+
+    def post(self, request):
+        """Update current authenticated user"""
+        try:
+            data = json.loads(request.body)
+        except json.JSONDecodeError:
+            data = request.data
+        return UpdatePersonController(
+            request.user, data, self.serializer_class
+        ).process()
+
+    def delete(self, request):
+        """Delete current authenticated user"""
+        try:
+            data = json.loads(request.body)
+        except json.JSONDecodeError:
+            data = request.data
+        return DeletePersonController(
+            request.user, data, self.serializer_class
+        ).process()
```

Comment:
you might want to think about implementing "soft delete" for users, especially if they are involved with many-to-many relationships such as rides etc. on pear we had some issues when users got deleted affecting their past matches so we had to [implement soft delete](https://github.com/cuappdev/pear-django-backend/pull/69), just a thought for u